### ISSUE TITLE
Notification suddenly stopped working

### ISSUE THREAD

**Issue by sxe** [rootkiwi/an2linuxserver#54]
Hi there, 

I know the title is kinda stupid but that's really what happened.
Since today I get no more notification and until today it worked fine for month.
Not even the test notification.

I did not install any updates on my phone nor my PC since then so I am at a loss about the reasons this could happen. 

I use an2linux on archlinux, latest git version. 

Is there any chance I can debug this somehow?

Cheers

**Comment by sxe** [comment]
I had some time and did some digging. 

Pairing works but notifications are still not showing up.
When I add `logging.info('TCPHandler > handle')` at https://github.com/rootkiwi/an2linuxserver/blob/master/an2linuxserver.py#L103 I get an output when pairing but not when a test notification is triggered from the android app. My python knowledge is pretty basic but this seems to show that the notification is not even received by the sever. Please correct me if I am wrong. I sadly cannot do any debugging on the android side, let me know if I can help any further.

Cheers

**Comment by sxe** [comment]
Yeah well, because I like talking to myself I will continue to do so. ;)
As a last resort I tried to restart my phone, like the good old windows days and what can I say.. it worked. Notifications work again!! I have no idea what could have caused this. I tried everything before. Reinstalling, deleting the app data, generating a new key pair..

Well all good now, maybe this helps someone in the future and it wasn't a complete wast of time. :)

Cheers
